cves/CVE-2003-0253.yml

```diff
@@ -103,11 +107,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: | 
+    The problem was reported by Saheed Akhtar. No mention on how it was found.
```

Comment:
Is there any information on where it was found? like if it was posted in some thread somewhere at anypoint? or does that fall under "no mention on how it was found"